addons/binding/org.openhab.binding.opendaikin/README.md

```diff
@@ -0,0 +1,62 @@
+# OpenDaikin Binding
+
+The OpenDaikin binding allows you to control your Daikin air conditioning units with OpenHAB. In order to do so, your Daikin air conditioning unit must have a BRP072A42 WiFi adapter installed.
+
+## Supported Things
+
+Daikin air conditioning units with a BRP072A42 installed. This may work with the older KRP series of wired adapters, but has not been tested with them.
+
+## Discovery
+
+This addon does not currently support discovery. You'll need to know the IP address of your air conditioner to add it as a thing.
+
+
+## Thing Configuration
+
+* host - The hostname of the Daikin air conditioner. Typically you'd use an IP address such as `192.168.0.5` for this field.
+* refresh - The frequency with which to refresh information from the Daikin air conditioner specified in milliseconds. Defaults to 60 seconds.
+
+
+## Channels
+
+* power - Turns the power on/off for the air conditioning unit.
+* settempc - The temperature set for the air conditioning unit in degrees Celsius. Can only be changed by half degree increments.
```

Comment:
Instead of separate channels for fahrenheit and celcius it is more common to use a configuration param on the thing or per channel to set the unit. See other bindings for examples.